control/summary_table.go

```diff
@@ -71,27 +73,104 @@ func (st *SummaryTable) Fill(openControlData opencontrols.Data) (err error) {
 	return
 }
 
+// diffControlOrigination computes the diff of the control origination.
+func (st *SummaryTable) diffControlOrigination(control string,
+	openControlData opencontrols.Data) ([]reporter.Reporter, error) {
+	// find the control origination section in the document.
+	docControlOriginationData, err := newControlOrigination(st)
+	if err != nil {
+		return nil, err
+	}
+	// find the control origins currently checked in the section in the doc.
+	docControlOrigins := docControlOriginationData.getCheckedOrigins()
+
+	// find the control origins noted in the yaml.
+	yamlControlOriginationData := openControlData.GetControlOrigins(control)
+	// find the control origins currently checked in the section in the YAML.
+	yamlControlOrigins := getCheckedOriginsFromYAML(yamlControlOriginationData)
```

Comment:
Since we need to do this same thing on the origins being returned from the `SummaryTable` _and_ the `yamlControlOriginationData`, maybe that should be its own type with a `GetChecked()` method?